bootstrap/Hemlock.opam

```diff
@@ -10,8 +10,8 @@ expressive type and module systems of [OCaml](http://ocaml.org/), a unified
 parametric effects/mutability/type system, and the shared-nothing
 message-passing parallelism of [Erlang](https://erlang.org/).
 """
-maintainer: "Jason Evans <je@branchtaken.net>"
-authors: ["Jason Evans"]
+maintainer: "Jason Evans <je@branchtaken.com>"
+authors: ["Jason Evans" "Cameron Evans"]
```

Comment:
Looked odd without the comma, but seems like the spec says this is correct. I'm assuming this passes `opam lint`.

Reply:
I didn't recall the existence of `opam lint`, but indeed this passes. :wink: